components/CollectivesWithData.js

```diff
@@ -73,52 +74,28 @@ class CollectivesWithData extends React.Component {
 
     const limit = this.props.limit || COLLECTIVE_CARDS_PER_PAGE * 2;
     return (
-      <div className="CollectivesContainer" ref={node => (this.node = node)}>
-        <style jsx>
-          {`
-            :global(.loadMoreBtn) {
-              margin: 1rem;
-              text-align: center;
-            }
-            .filter {
-              width: 100%;
-              max-width: 400px;
-              margin: 0 auto;
-            }
-            :global(.filterBtnGroup) {
-              width: 100%;
-            }
-            :global(.filterBtn) {
-              width: 33%;
-            }
-            .Collectives {
-              display: flex;
-              flex-wrap: wrap;
-              flex-direction: row;
-              justify-content: center;
-              overflow: hidden;
-              margin: 1rem 0;
-            }
-            .CollectivesContainer :global(.CollectiveCard) {
-              margin: 1rem;
-            }
-          `}
-        </style>
-
-        <div className="Collectives cardsList">
+      <Container ref={node => (this.node = node)}>
+        <Container
+          display="flex"
+          flexWrap="wrap"
+          flexDirection="row"
+          justifyContent="center"
+          overflow="hidden"
+          margin="1rem 0"
+        >
           {collectives.map(collective => (
-            <CollectiveCard key={collective.id} collective={collective} />
+            <CollectiveCard style={{ margin: '1rem' }} key={collective.id} collective={collective} />
```

Comment:
I'm not sure that `CollectiveCard` accepts this prop. Could be nice to migrate `CollectiveCard` in the same PR as both components are closely tied.

Reply:
I've corrected that and also migrated CollectiveCard in this PR. Feel free to let me know if anything further needs to be done. :smile: